examples/specs/layer_bar_dual_axis_minmax.vl.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
So what's your goal here?  

If you want it as an example in the gallery, I don't think it's a good example to include in the gallery.  Even if you are okay with dual axes, a better design would be to use `area` band rather than two lines anyway. 

If you want a test case for layer of layer, then layer_box_plot_circle already serves the purpose and is less controversial example.